src/node/utility/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "knightcrawler-utility",
```

Comment:
Alternatively you can add a [scope](https://docs.npmjs.com/cli/v7/using-npm/scope) like `@knightcrawler/utility`, the same style applies to the other packages in the format `@project/package`. Though it's not really required as just `utilities` works too.